utils/date-constructor.js

```diff
@@ -0,0 +1,41 @@
+const fullDate = new Date();
+
+const weekDay = [
+  "Sunday",
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+];
+
+const month = [
+  "January",
+  "February",
+  "March",
+  "April",
+  "May",
+  "June",
+  "July",
+  "August",
+  "September",
+  "October",
+  "November",
+  "December",
+];
+
+const date =
+  weekDay[fullDate.getDay()] +
+  ", " +
+  fullDate.getDate() +
+  ". " +
+  month[fullDate.getMonth()] +
+  " " +
+  fullDate.getFullYear() +
+  ", " +
+  fullDate.getHours() +
+  ":" +
+  fullDate.getMinutes();
```

Comment:
I assume that your aim is to record the exact time the entry was created.  
⚠️ _If so, please check whether this is the case._

In your code `date` is a predefined variable, it captured the date and time at the moment your application starts or the module is first imported. This means that every entry you created would use this same timestamp, regardless of when the entry was actually added. 

The solution could be to converting `date` to a function like `getCurrentFormattedDate` which computes the current date and time upon each call, you ensure that every new entry gets the exact date and time of when that entry was made and use this function **when creating new entries.**

Reply:
Hi Jessica, thank you for your review. In our tests, this feature worked as intended, i. e. each entry got its own timestamp depending on the time in the moment of submitting the form.  That's most likely caused by using `new Date()` and the built-in javascript methods do construct our date and time in our date-constructor.js. So there is a function included, computing the _current_ data.